Source/Fuse.Controls.Panels/LayoutControl.uno

```diff
@@ -193,5 +193,16 @@ namespace Fuse.Controls
 				return LayoutDependent.Maybe;
 			}
 		}
+
+		protected override bool FastTrackDrawWithOpacity(DrawContext dc)
+		{
+			if (HasChildren) return false;
+			
+			if (Background != null)
+				DrawBackground(dc, Opacity);
+			
+			// Asserting base class doesn't need to draw anything!
```

Comment:
The wording's a bit weird, but it means it's intentionally not calling the base class since shouldn't have anything to do.